config/routes.rb

```diff
@@ -209,6 +209,8 @@
   get "health_checks/enqueue_failing_job", to: "health_checks#enqueue_failing_job"
   root "welcome#index"
 
+  get "/prendre-rdv", to: "search#prendre_rdv"
```

Comment:
L’action peut quand même s’appeler `search_rdv`, ça me fait quand même bizarre d’avoir ça en français. 🤔 

```suggestion
  get "/prendre-rdv", to: "search#seach_rdv"
```

Reply:
Ah tiens, d’ailleurs dans le reste de l’app on a laissé des paths avec des underscores. Je suis partagé entre l’envie de mettre un underscore ici aussi par cohérence et celle de mettre des tirets partout. 🤷

Reply:
Oui je suis vraiment pas fan non plus d'avoir du français, mais on part déjà de loin sur cette codebase 😄 . C'était aussi pour rester cohérent avec l'url helper `pendre_rdv_path`.
J'ai changé du coup, et j'ai mis l'underscore dans l'url par souci de cohérence même si j'aime moins.